app/build.gradle

```diff
@@ -1,9 +1,11 @@
 import java.text.SimpleDateFormat
 
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-kapt'
-apply plugin: "androidx.navigation.safeargs.kotlin"
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'kotlin-kapt'
+    id 'androidx.navigation.safeargs.kotlin'
```

Comment:
Removing `androidx.navigation.safeargs.kotlin` doesn't break the build. Does this mean that we're not using this and might as well remove it? @sasho-axellero

Reply:
It is required for the Navigation component below, but that is also not used.
The whole file should be cleaned up and libraries updated.
I don't know if this is something we should do with this update.

Reply:
In context of improving Gradle, I think it's fine "boyscouting" and removing dead code.
Timebox it to 1-2 hours please.

Reply:
If it's something minor, please go ahead.

Reply:
Ok, I'll clean and update